zfs-replications-ubuntu.sh

```
@@ -111,10 +121,29 @@ send_notification() {
+            --argjson priority "$priority" \
+            '{title: $title, message: $message, priority: $priority}')
+        
+        local response
+        response=$(curl -s -X POST "$GOTIFY_SERVER_URL/message" \
             -H "Content-Type: application/json" \
-            -d "{\"title\":\"$title\",\"message\":\"$message\",\"priority\":$priority}" \
-            -H "X-Gotify-Key: $GOTIFY_APP_TOKEN" >/dev/null 2>&1
+            -d "$json_payload" \
+            -H "X-Gotify-Key: $GOTIFY_APP_TOKEN" 2>&1)
+        
+        if [[ $? -eq 0 ]]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)